src/PackageManagement.VisualStudio/Utility/EnvDTEProjectUtility.cs

```diff
@@ -625,7 +640,9 @@ public static string GetTargetFrameworkString(EnvDTEProject envDTEProject)
                 ||
                 string.Equals(envDTEProject.Kind, NuGetVSConstants.FsharpProjectTypeGuid, StringComparison.OrdinalIgnoreCase)
                 ||
-                string.Equals(envDTEProject.Kind, NuGetVSConstants.JsProjectTypeGuid, StringComparison.OrdinalIgnoreCase))
+                string.Equals(envDTEProject.Kind, NuGetVSConstants.JsProjectTypeGuid, StringComparison.OrdinalIgnoreCase)
+                ||
+                string.Equals(envDTEProject.Kind, NuGetVSConstants.CpsProjectTypeGuid, StringComparison.OrdinalIgnoreCase))
```

Comment:
Inappropriate fix, since it will only fix msbuildproj and not all other CPS-based project types as required.
This is why it's a good idea to delete the `NuGetVSConstants.CpsProjectTypeGuid` constant -- to make it harder to accidentally hook behavior to a file extension instead of project capabilities.